src/Flight/FlightController.java

```diff
@@ -26,52 +28,55 @@ public void getFlightInfo() {
 
-            System.out.println("Enter destination airport three letter code (Ex. PHL): ");
-            String destinationAirport = scanner.next();
+        System.out.println("Enter origin airport three letter code (Ex. PHL): ");
+        String originAirport = scanner.next();
 
-            System.out.println("Enter number of seats: ");
-            // Generally, when you cast a value you should only cast smaller to larger
-            short seatCount = (short) scanner.nextInt(); // taking the int and converting it's datatype to short
+        System.out.println("Enter destination airport three letter code (Ex. PHL): ");
+        String destinationAirport = scanner.next();
 
-            flightToAdd = new Flight(flightNumber, originAirport, destinationAirport, seatCount);
+        System.out.println("Enter number of seats: ");
+        // Generally, when you cast a value you should only cast smaller to larger
+        short seatCount = (short) scanner.nextInt(); // taking the int and converting it's datatype to short
 
-            flights[indexToReplace] = flightToAdd;
-            System.out.printf("Flight %s successfully added\n", flightToAdd);
-        }
+        flightToAdd = new Flight(flightNumber, originAirport, destinationAirport, seatCount);
+        flightService.createFlight(flightToAdd);
     }
-
     // TODO: Implement a method to update flight information by ID
     public void updateFlightInformation(){
+        Flight flightToUpdate;
 
-    }
+        System.out.println("Please enter flight to update, starting with Flight Number: ");
+        int flightNumber = scanner.nextInt();
+        scanner.nextLine();
 
-    public static boolean isEmpty(Flight[] arr) { // defining the parameter of a string array to be included when executing this mehtod
-        for (Flight element : arr) { // enhanced for each loop, that iterates through the arrays elements returning & assigning the value to the declared variable flight
-            if (element != null) {
-                return false;
-            }
+        flightToUpdate = flightService.findById(flightNumber);
+        if(flightToUpdate == null) {
+            System.out.println("Flight not found");
+            return;
         }
-        return true;
-    }
 
-    public int getFirstNull(Flight[] arr) { //get the first null index to replace, otherwise return -1
-        for (int i = 0; i < arr.length; i++) {
-            if (arr[i] == null) {
-                return i;
-            }
-        }
-        return -1;
+        System.out.println("Enter time of departure date & time formatted as \"2024-07-03 15:48:00\" ");
+        String timeDeparture = scanner.nextLine();
+
+        System.out.println("Enter time of arrival date & time formatted as \"2024-07-03 15:48:00\" ");
+        String timeArrival = scanner.nextLine();
+
+        System.out.println("Enter pilot ID: ");
+        int pilot = scanner.nextInt();
+        scanner.nextLine();
```

Comment:
I've seen a few cases where you use nextLine() after nextInt to move on to the next line and other cases where you use next() instead, wondering what the pros and cons of using each method are?